predict_loading_concentration.py

```diff
@@ -4,35 +4,37 @@ def warn(*args, **kwargs):
 import warnings
 warnings.warn = warn
 
-from diva_dna_seq.utility import aproximate_sequence,prepare_data
-from diva_dna_seq import predict_loading_concentration
+from diva_seq_opt import utility
 import argparse
 import pickle
 
 
+def main():
+	#Allow Matplotlib to be used on commandline
+	import matplotlib as mpl
+	mpl.use('Agg')
 
-#Allow Matplotlib to be used on commandline
-import matplotlib as mpl
-mpl.use('Agg')
+	#Handle inputs
+	#Parse Inputs
+	parser = argparse.ArgumentParser(description='Use BioAnalyzer Data to Predict Library Loading Concentration for Library Barcoding')
 
-#Handle inputs
-#Parse Inputs
-parser = argparse.ArgumentParser(description='Use BioAnalyzer Data to Predict Library Loading Concentration for Library Barcoding')
+	parser.add_argument('BAF',type=str, help='BioAnalyzer CSV')
+	parser.add_argument('LAD',type=str, help='Ladder CSV')
+	parser.add_argument('OF' ,type=str, help='Prediction Plot output file, *.png')
+	args = parser.parse_args()
 
-parser.add_argument('BAF',type=str, help='BioAnalyzer CSV')
-parser.add_argument('LAD',type=str, help='Ladder CSV')
-parser.add_argument('OF' ,type=str, help='Prediction Plot output file, *.png')
-args = parser.parse_args()
+	#Load Model
+	with open('./diva_seq_opt/model/model30.pkl','rb') as fp:
```

Comment:
Later on we should change this model name to e.g. model_latest.pkl and always have a copy of the latest model. For example, at the end of the year the `model` directory will contain:
model_2018.pkl, model_05_2021.pkl, model_12_2021.pkl, model_latest.pkl, where the last two would be the same.

Reply:
I very much agree for the future!

Reply:
Can you please include a TODO about that before this line?

Reply:
I think the path for models should not be among the code but in a separate directory at the same level as notebooks.